packages/mdc-checkbox/_mixins.scss

```diff
@@ -518,7 +521,7 @@
   $feat-structure: mdc-feature-create-target($query, structure);
 
   @include mdc-feature-targets($feat-structure) {
-    transform: scale(2.75, 2.75);
+    transform: scale(1);
```

Comment:
can you remove scale here? I don't think this is being animated, so you don't need it.